lib/src/configuration.dart

```diff
@@ -241,6 +242,14 @@ extension FlexibleSyncConfigurationInternal on FlexibleSyncConfiguration {
   set sessionStopPolicy(SessionStopPolicy value) => _sessionStopPolicy = value;
 }
 
+class SyncFollowerConfiguration extends Configuration {
+  SyncFollowerConfiguration._(
+    super.schemaObjects, {
+    super.fifoFilesFallbackPath,
+    super.path,
```

Comment:
Isn't it dangerous to change the paths after the realm already exists and it is synced?

Reply:
Do you have some test sample that display this scenario with opening local realm?

Reply:
The path needs to match the one used, when opening the same realm with a `FlexibleSyncConfiguration`

Reply:
I will add a test.

Reply:
No I won't .. it will require some refactoring of the test harnish